notebooks/mos_spectroscopy_advanced/MOSspec_advanced.ipynb

```diff
@@ -0,0 +1,1060 @@
+{
```

Comment:
<p>Running this cell results in&nbsp;<strong>FileNotFound Error</strong>.As the above cell file1d and file2d reference the&nbsp;<strong>mos_ceers_data</strong>&nbsp;folder in the relative directory, which doesn't exist.</p><p> </p><pre class="ql-syntax" spellcheck="false">FileNotFoundError: Could not locate file: mos_ceers_data/jw01345-o064_s02904_nirspec_f100lp-g140m_s2d.fits
</pre><p> </p><p>The temp folder is where all .fits files live under the <strong>data</strong> folder. So changing them to, for instance, file1d = "./data/" +"jw013..." would be one way to fix it.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/117/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Should be fixed now.